tests/test_on_data/test_gin_ecephys.py

```diff
@@ -368,12 +368,14 @@ class TestConverter(NWBConverter):
                     DATA_PATH / "cellexplorer" / "dataset_1" / "20170311_684um_2088um_170311_134350.spikes.cellinfo.mat"
                 )
             ),
+            case_name="CE1",
         ),
         param(
             data_interface=CellExplorerSortingInterface,
             interface_kwargs=dict(
                 file_path=str(DATA_PATH / "cellexplorer" / "dataset_2" / "20170504_396um_0um_merge.spikes.cellinfo.mat")
             ),
+            case_name="CE2",
```

Comment:
This ends up being necessary to avoid file access collisions (file name depends on unique `case_name` when testing the same data interface multiple times)